### ISSUE TITLE
E5108

### ISSUE THREAD

**Issue by PoorThatcher** [jakewvincent/mkdnflow.nvim#53]
`E5108: Error executing lua ...nfig/nvim/plugged/mkdnflow.nvim/lua/mkdnflow/buffers.lua:52: attempt to concatenate local 'prev_buf' (a nil value)
stack traceback:
        ...nfig/nvim/plugged/mkdnflow.nvim/lua/mkdnflow/buffers.lua:52: in function 'goBack'
        [string ":lua"]:1: in main chunk`

use goback error

When I try to goback in first link & press again <BS>

**Comment by PoorThatcher** [comment]
When I go forward, everything is fine
`⬇️  Can't go forward any further!`

**Comment by jakewvincent** [comment]
Is this happening regularly, every time you can't go back any further? If not, can you summarize when this happens?
